lib/statesman/adapters/active_record_transition.rb

```diff
@@ -3,8 +3,20 @@
 module Statesman
   module Adapters
     module ActiveRecordTransition
-      def self.included(base)
-        base.send(:serialize, :metadata, JSON)
+      DEFAULT_UPDATED_TIMESTAMP_COLUMN = :updated_at
+
+      extend ActiveSupport::Concern
+
+      included do
+        serialize :metadata, JSON
+
+        class_attribute :updated_timestamp_column
+
+        def self.reset_updated_timestamp_column
```

Comment:
I don't really understand why a user ever need / want this? If they do `updated_timestamp_column = :foo` they can just remove the line to use the default column right?

Reply:
I have this primarily to provide a way to reset the setting to its default state in tests, without depending on knowing the setting name and the correct default. If people don't like this, I could just refer to `DEFAULT_UPDATED_TIMESTAMP_COLUMN`.

Reply:
+1 for getting rid of this unwanted method.